docs/built-in-rules/index.mdx

```diff
@@ -7,9 +7,32 @@ import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 import styles from "./styles.module.scss";
 import BrowserCustomTabs from "@site/src/components/built-in-rules/CustomTabs.js";
+import PolicyCard from "@site/src/components/built-in-rules/PolicyCard.js";
+import Grid from "@mui/material/Grid";
+
 
 # Built in rules
 
-There are dozens of battle-tested rules for you to choose from, that cover different Kubernetes resources and use cases:
+Datree comes with dozens of battle-tested rules that cover different Kubernetes resources and use cases.  
+The rules are also grouped into the following policies which you can use out-of-the-box:
 
-<BrowserCustomTabs />
+<Grid container spacing={1.5}>
+    <Grid item>
```

Comment:
You can clean up here a little bit by using loop
[https://bobbyhadz.com/blog/react-loop-through-object](https://bobbyhadz.com/blog/react-loop-through-object)